server/controllers/profile.js

```diff
@@ -38,4 +36,35 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route GET /profile/sitter
+// @desc Get all profile data
+// @access Private
+exports.getAllSitters = asyncHandler(async (req, res) => {
+  const result = [];
+  const profiles = await Profile.find({ isSitter: true, price: { $ne: null } })
+  if (profiles) {
+    profiles.filter((profile) => {
```

Comment:
Nitpick! The use of `filter` method here is redundant. Maybe `forEach` is what we need to use.

Reply:
change filter to forEach